app/assets/stylesheets/landing.css.scss

```diff
@@ -0,0 +1,5 @@
+
+#main-logo {
+	height: 100%;
+	background-color: orange;
```

Comment:
Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs